src/error.rs

```diff
@@ -166,7 +167,15 @@ u32_enum_builder! {
         CertSCTInvalidSignature => 7320,
         CertSCTTimestampInFuture => 7321,
         CertSCTUnsupportedVersion => 7322,
-        CertSCTUnknownLog => 7323
+        CertSCTUnknownLog => 7323,
+
+        // From InvalidCertRevocationList
+        // https://docs.rs/rustls/0.21.3/rustls/enum.Error.html#variant.InvalidCertRevocationList
+        //
+        // TODO(XXX): due to a small oversight the 0.21.3 release did not export the
+        //   InvalidCertRevocationList's CertRevocationListError enum. Once this is fixed + released
+        //   upstream we should flatten the sub-variants here.
```

Comment:
Fumbling through a maintenance release in https://github.com/rustls/rustls/pull/1348 to hopefully get this sorted before merge.

Reply:
This was fixed in [v0.21.4](https://github.com/rustls/rustls/releases/tag/v%2F0.21.4) - I will update this branch to provide better error specificity using that fix shortly.